patch/kernel/archive/rockchip64-5.10/linux-90114-add-rk3399-roc-pc-plus-ir-keymap.patch

```diff
@@ -0,0 +1,58 @@
+--- /dev/null
```

Comment:
This and next two patches would best be combined into a single one.
It should also be renamed as it is not rk3399-roc-pc-plus specific as it also covers rk3328-roc-cc and other station boards' remotes.